servicebus/data-plane/.gitignore

```diff
@@ -0,0 +1,53 @@
+*.class
```

Comment:
Can you compare this to the one at root? I have a feeling the one at root is a superset of this and it is nto necessary.